model/domain/searchresult.go

```diff
@@ -43,8 +47,8 @@ func (sr *SearchResult) UnmarshalProto(searchResultAsBytes []byte) error {
 		return err
 	}
 
-	sr.ToDomain(pbSearchResult)
-	return nil
+	err := sr.ToDomain(pbSearchResult)
```

Comment:
return sr.ToDomain(pbSearchResult)

Reply:
This style is simply to gives a name to what is returned.